Bottom: b7908a5c682466249184ca18ad389fa1b574c3b9
Top:    e59a8c8a92560ce29975049974660fa4c31bbe33
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-10-24 18:03:15 -0700

Refresh of riostream

---

diff --git a/examples/riostream.c b/examples/riostream.c
index 7333a3d..dfb03e5 100644
--- a/examples/riostream.c
+++ b/examples/riostream.c
@@ -291,7 +291,7 @@ static int send_xfer(int size)
 		if (ret > 0) {
 			offset += ret;
 		} else if (errno != EWOULDBLOCK && errno != EAGAIN) {
-			perror("rsend");
+			perror("riowrite");
 			return ret;
 		}
 	}
@@ -371,6 +371,7 @@ static int run_test(void)
 	*poll_byte = -1;
 	offset = riomap(rs, buf, transfer_size, PROT_WRITE, 0, 0);
 	if (offset ==  -1) {
+		perror("riomap");
 		ret = -1;
 		goto out;
 	}
